### PR TITLE
config: use member ORCID api

### LINKED ISSUE #2235: orcid: use member API by default
Follow-up of https://twitter.com/inspirehep/status/852454184704237568: we patched the issue in production, but we still need to integrate the fix in `master`.

### PR DESCRIPTION
**Pull request by david-caro**
closes #2235

Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

File: inspirehep/config.py [hunk 1]
```diff
@@ -1320,9 +1320,17 @@ def _(x):
 
 # OAuthclient
 # ===========
-orcid.REMOTE_APP['params']['request_token_params'] = {
-    'scope': '/orcid-profile/read-limited /activities/update /orcid-bio/update',
-    'show_login': 'true'}
+orcid.REMOTE_APP['params'].update({
+    'access_token_url': 'https://orcid.org/oauth/token',
+    'request_token_params': {
+        'scope': ' '.join(
```
Comment (jacquerie) [inline inspirehep/config.py]: (As Travis complained, `.join` takes a list, not `*args`)

**Comment by jacquerie** [comment]
We could use the `REMOTE_MEMBER_APP` instead. It's basically the same as this, except that `base_url` is also set: https://github.com/inveniosoftware/invenio-oauthclient/blob/7e8cd48f20a07ec97337650dda272a469c285c7a/invenio_oauthclient/contrib/orcid.py#L121-L127 (ignore line 128, it's a mistake).

**Comment by david-caro** [comment]
I've already done the change on gitlab.

**Comment by jacquerie** [comment]
> I've already done the change on gitlab.

(Does GitLab support linking the file state at a certain SHA, so that links do not go stale?)

**Comment by jacquerie** [comment]
Rebase and :shipit:

**Comment by david-caro** [comment]
> (Does GitLab support linking the file state at a certain SHA, so that links do not go stale?)

https://gitlab.cern.ch/ai/it-puppet-hostgroup-inspire/blob/33e02d5cfc0516d199e86744704d67df3f01264f/code/templates/inspire/invenio.production_inv3.cfg#L90

yep :)

**Comment by david-caro** [comment]
Not sure though if you reference a stale commit (that was removed) if it will be available for a long time.

**Pull request merged**